CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+
+### Joining Us!
+
+1. We are a TC39 task group! To contribute, you'll need to be added either as a TC39 member or an invited expert. If you are already a member, you can find our meetings on the TC39 calendar. If you're interested in becoming a member, or would like to be added as an invited expert, join our Matrix room (link below) and message "jkup".
+2. We use Matrix for our chat rooms. You can find our main room here: https://matrix.to/#/#tc39-tg4:matrix.org
+
+### Documents
+
+- [Specification Repository](https://github.com/tc39/source-map-spec)
+- [RFC discussions](https://github.com/tc39/source-map-rfc)
+- [Our website](https://source-map.github.io/)
+- [Source map validator (very much work in progress)](https://github.com/jkup/source-map-validator)
+- [Stacktrace validator (very much work in progress)](https://github.com/jkup/source-map-stacktrace-validator)
+
+### Workstreams
+
+- **Meetings**. We currently have 2 different recurring meetings.
+  - Public feature meeting. Open to all TC39 members, this is where we talk about new features we'd like to add to the specification. They can be found under the [naming](https://github.com/tc39/source-map-rfc/issues?q=is%3Aopen+is%3Aissue+label%3A%22Workstream%3A+Naming%22) label on GitHub.
+  - Public general meeting. Open to all TC39 members, this is where we talk about errors or vague bits of the specification. We discuss issues found under the [correctness](https://github.com/tc39/source-map-rfc/issues?q=is%3Aopen+is%3Aissue+label%3A%22Workstream%3A+Correctness%22+) label on GitHub.
+- **Testing**. We're still early stages but this is one of the most important things to focus on. When we present our updates to the main TC39 plenary, we need to have browsers and tools using these tests in accordance with our [process document](https://github.com/tc39/source-map-rfc/blob/main/PROCESS.md).
+
+#### Customers
+
+We've been thinking about "source map customers" in the following terms. Please feel free to suggest better titles or categorization methods!
+
+1. Generators. Bundlers, transpilers, compilers. Tools that have access to the source code and emit (among other things) source maps.
+2. Debuggers. Browsers and [stand alone debuggers](https://www.replay.io/). These are tools that have access to the source map and the generated code.
+3. Post-hoc debuggers. Tools like Sentry and New Relic. These tools have access to an error stack trace and a source map.
+
+#### Testing
+
+We've been categorizing our testing efforts into three groups, each matching with a "customer" group above.
+
+1. Validators. For testing generator tools, we'd like to have the ability to validate a source map on it's own as well as a source map being a valid map between a source and a generated file. We did a mini hackathon day on what this might look like: https://github.com/jkup/source-map-validator
```

Comment:
```suggestion
1. Validators. For testing generator tools, we'd like to have the ability to validate a source map on its own as well as a source map being a valid map between a source and a generated file. We did a mini hackathon day on what this might look like: https://github.com/jkup/source-map-validator
```